content/influxdb/v2.2/write-data/troubleshoot.md

```diff
@@ -60,25 +77,240 @@ Write requests return the following status codes:
+- Minimize payload size and network errors by [optimizing writes](/influxdb/v2.2/write-data/best-practices/optimize-writes/).
 
-### Troubleshoot rejected points
+{{% /oss-only %}}
 
-InfluxDB may reject points even if the HTTP request returns "Success".
-If some of your data did not write to the bucket, check for [field type](/influxdb/v2.2/reference/key-concepts/data-elements/#field-value) differences between the missing data point and other points that have the same [series](/influxdb/v2.2/reference/key-concepts/data-elements/#series).
-For example, did you attempt to write `string` data to an `int` field?
+{{% cloud-only %}}
+If you notice data is missing in your bucket, do the following:
+
+- Check the `message` property in the response body for details about the error--for example, `partial write error` indicates [rejected points](#troubleshoot-rejected-points).
+- Check for [rejected points](#troubleshoot-rejected-points) in your organization's `_monitoring` bucket.
+- Verify all lines contain valid syntax ([line protocol](/influxdb/cloud/reference/syntax/line-protocol/) or [CSV](/influxdb/cloud/reference/syntax/annotated-csv/)). See how to [find parsing errors](#find-parsing-errors).
+- Verify the data types match the [series](/influxdb/cloud/reference/key-concepts/data-elements/#series) or [bucket schema](/influxdb/cloud/organizations/buckets/bucket-schema/). See how to resolve [explicit schema rejections](#resolve-explicit-schema-rejections).
+- Verify the timestamps match the [precision parameter](/influxdb/cloud/write-data/#timestamp-precision).
+- Minimize payload size and network errors by [optimizing writes](/influxdb/cloud/write-data/best-practices/optimize-writes/).
+{{% /cloud-only %}}
+
+## Troubleshoot rejected points
+
+{{% oss-only %}}
 
 InfluxDB rejects points for the following reasons:
+
 - The **batch** contains another point with the same series, but one of the fields has a different value type.
 - The **bucket** contains another point with the same series, but one of the fields has a different value type.
+
+Check for [field type](/influxdb/v2.2/reference/key-concepts/data-elements/#field-value) differences between the missing data point and other points that have the same [series](/influxdb/v2.2/reference/key-concepts/data-elements/#series)--for example, did you attempt to write `string` data to an `int` field?
+
+{{% /oss-only %}}
+
+{{% cloud-only %}}
+
+InfluxDB may have rejected points even if the HTTP request returned "Success".
+InfluxDB logs rejected data points and associated errors to your organization's `_monitoring` bucket.
+
+- [Review rejected points](#review-rejected-points)
+  - [Find parsing errors](#find-parsing-errors)
+  - [Find data type conflicts and schema rejections](#find-data-type-conflicts-and-schema-rejections)
+- [Resolve data type conflicts](#resolve-data-type-conflicts)
+- [Resolve explicit schema rejections](#resolve-explicit-schema-rejections)
+
+### Review rejected points
```

Comment:
Maybe be nice to add example that shows an example query that shows how to find rejected points/.

And/or clarify parsing errors and data type conflicts/schema rejections are a subset of rejected points?